templates/agent.containerd.yaml

```diff
@@ -22,6 +22,16 @@ identities:
     dns_pattern: .*.gcr.io
     dst_port: 443
   pushdown: []
+- node:
+    name: ghcr_io
+    dns_pattern: ghcr.io
+    dst_port: 443
+  pushdown: []
+- node:
+    name: pkg_containers_githubusercontent
```

Comment:
usually we name it in reverse order: githubusercontent_containers_pkg

Reason being: it will sort together with all other links going to githubusercontent